src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -18,11 +19,12 @@
 CONFIG_URL = 'https://hg.mozilla.org/mozilla-central/raw-file/tip/tools/clang-tidy/config.yaml'
 REPO_CENTRAL = b'https://hg.mozilla.org/mozilla-central'
 REPO_UNIFIED = b'https://hg.mozilla.org/mozilla-unified'
-ARTIFACT_URL = 'https://queue.taskcluster.net/v1/task/{task_id}/runs/{run_id}/artifacts/public/results/{diff_name}'
 
 
 logger = get_logger(__name__)
 
+TaskCluster = collections.namedtuple('TaskCluster', 'results_dir, task_id, run_id, online')
```

Comment:
Nit: `online` is not a great name, but I don't have another idea unless we make it the opposite and call it `local`